qiskit/algorithms/phase_estimators/hamiltonian_phase_estimation_result.py

```diff
@@ -81,6 +82,17 @@ def filter_phases(self, cutoff: float = 0.0, scaled: bool = True,
         else:
             return cast(Dict, phases)
 
+    @property
+    @deprecate_function('The HamiltonianPhaseEstimationResult.most_likely_phase attribute '
```

Comment:
Could you use our standard text to include when it's been deprecated etc.? 🙂 
```python
import warnings

def foo(input):
    warnings.warn('The qiskit.foo() function is deprecated as of 0.9.0, and '
                  'will be removed no earlier than 3 months after that '
                  'release date. You should use the qiskit.bar() function '
                  'instead.', DeprecationWarning, stacklevel=2)

```
(see https://qiskit.org/documentation/contributing_to_qiskit.html#deprecation-warnings)

Reply:
Ok. Btw, I modeled this on deprecated functions in quantumcircuit.py. According to the doc you link above, `qiskit.utils.deprecation` should never be used?